### PR TITLE
[Merged by Bors] - Rename smartmodule arguments, update help

### LINKED ISSUE #67: Rename instances of SmartStream to SmartModule

### PR DESCRIPTION
**Pull request by nicholastmosher**
Closes #67

### CODE REVIEW

File: rust-connectors/common/src/opt.rs [hunk 1]
```diff
@@ -18,23 +20,26 @@ pub struct CommonSourceOpt {
...
 
-    /// Path of map smartstream used as a pre-produce step
-    /// If not found file in that path, it will be fetch
-    /// the smartmodule with that name if present
+    /// Path of map smartmodule used as a pre-produce step
+    ///
+    /// If the value is not a path to a file, it will be used
+    /// to lookup a SmartModule by name
     #[structopt(long, group("smartstream"))]
-    pub smartstream_map: Option<String>,
```
Comment (simlay) [inline rust-connectors/common/src/opt.rs:31]: please rename the structopt group.

**Review by simlay — CHANGES_REQUESTED** [review]
I think the integration CI will fail. You should update the [yaml in the http connectoctor](https://github.com/infinyon/fluvio-connectors/blob/82436ded12cb8dc8ed963b7cbe93e9a66474b57a/rust-connectors/sources/http/tests/get-smartstream-config.yaml#L13)

**Review by simlay — APPROVED** [review]

**Comment by nicholastmosher** [comment]
bors ping

**Pull request closed without merging**